Algorithms/DataCompression/huffman.cs

```diff
@@ -2,20 +2,21 @@
 using System.Collections.Generic;
 using System.Linq;
 
-namespace DC5
+namespace Algorithms.DataCompression
 {
     internal class Huff
```

Comment:
Why this class is called Huff? It looks more like ListNode.

Reply:
It represents a tree structure for the algorithm, the name can be changed, is not relevant.

Reply:
Please rename this class to ListNode